src/components/timepicker/timepicker.scss

```diff
@@ -58,12 +56,12 @@ $m-timepicker--color-active: $m-color--grey-lightest !default;
         &__hours,
         &__minutes {
             width: 50%;
-            flex: 1 0 auto;
             list-style: none;
             padding: 0;
             margin: 0;
             text-align: center;
-            overflow: auto;
+            overflow-y: scroll;
+            -webkit-overflow-scrolling: touch;
```

Comment:
This example has a listbox with more options than its height can accommodate.
Scrolling only works as expected if the listbox is the options' offsetParent. The example uses position: relative on the listbox to that effect.

Je pense que le overflow est p-e même pas nécéssaire